tests/fsharp/Compiler/Language/TypeDirectedConversionTests.fs

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
```

Comment:
More of a nitpick, but for new tests in future we probably prefer to have them in the ComponentTests suite, it's a bit easier to chain (i.e. baseline validation, IL validation, running, etc), and a bit easier to debug and configure. Not a big deal though.